feat: Service Catalog (#7889)
Kong/docs.konghq.com: app/konnect/service-catalog/integrations/pagerduty.md

---
title: PagerDuty Integration
content-type: reference
config:
    discovery_support: true
    discovery_default: true
    bindable_entities: "PagerDuty Service"
    mechanism: "pull/ingestion model"
---

_Type: External_

The PagerDuty integration allows you to provide a way to alert the service team (via PagerDuty services) as well as provide information on current open incidents to consumers of the service directory. 

> For each linked PagerDuty service, a summary should be provided showing current unresolved incidents and the current on-call user.

Reviewer: Should we instead say something like "...a summary **will** be provided **on the Service Catalog service's details page**, showing current unresolved incidents and the current on-call user."?

> For each linked PagerDuty service, a summary should be provided showing current unresolved incidents and the current on-call user.

Author: ```suggestion
For each linked PagerDuty service, a summary will be provided on the Service Catalog service's details page, showing current unresolved incidents and the current on-call user.
```

> An option to start an incident for the PagerDuty service

Reviewer: Think we need to complete this sentence.

> An option to start an incident for the PagerDuty service

Author: ```suggestion
```

## Authenticate the PagerDuty integration

1. From {{site.konnect_product_name}} select **[Integrations](https://cloud.konghq.com/us/service-catalog/integrations)**. 
2. Select **PagerDuty**, then **Install PagerDuty**.
3. Select **Authorize**. 

PagerDuty will ask you to grant consent to {{site.konnect_product_name}}. Both Read and Write scopes are required.

> ## Bindable entities

Reviewer: Could we call these Resources instead? "Bindable entities" is a term that our engineering team uses synonymously with Resources, and I think that we should just stick to the latter for customer-facing documentation.

> ## Bindable entities

Author: ```suggestion
## Resources
```

> Entity | Binding Level | Description
> -------|---------------|-------------
> PagerDuty Service | Service | A PagerDuty service is any entity that can have incidents opened on it. In practice it could be a service, but could also be a group of services or an organization/team.

Author: ```suggestion
Entity | Description
-------|------------
PagerDuty Service | A PagerDuty service is any entity that can have incidents opened on it. In practice it could be a service, but could also be a group of services or an organization/team.
```

## Discovery information

> {% include_cached service-catalog-discovery.html %}

Reviewer: I'm not entirely sure why I'm seeing "Repositories" appear as the bindable entity for PagerDuty, but that sounds incorrect.

<img width="2032" alt="Screenshot 2024-09-10 at 4 23 30 PM" src="https://github.com/user-attachments/assets/456cbe6e-6a23-442b-9c52-285e807cce07">
